src/content/en/updates/2020/11/cvd.md

```diff
@@ -0,0 +1,482 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Simulating color vision deficiencies in the Blink Renderer. The Chrome DevTools engineering blog - by the developers who build the DevTools.
+
+{# wf_updated_on: 2020-11-13 #}
+{# wf_published_on: 2020-11-13 #}
+{# wf_tags: devtools-blog #}
+{# wf_featured_image: /web/updates/images/2020/11/cvd/cvd.png #}
+{# wf_featured_snippet: Simulating color vision deficiencies in the Blink Renderer. #}
+{# wf_blink_components: N/A #}
+
+# Simulating color vision deficiencies in the Blink Renderer {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+
+This article describes why and how we implemented color vision deficiency simulation in DevTools and the Blink Renderer.
+**Note:** If you prefer watching a presentation over reading articles, then enjoy the video below! If not, skip the video and read on.
```

Comment:
Apparently I was overcomplicating things :D This is how to get the special note style to show up:

```suggestion

Note: If you prefer watching a presentation over reading articles, then enjoy the video below! If not, skip the video and read on.
```